### PR TITLE
reduce the lower-bound on the lightgbm dependency

### PR DESCRIPTION
**Pull request by imatiach-msft**
reduce lightgbm lower bound dependency to >=2.0.11, since raiwidgets, raitools and erroranalysis are all compatible with it

### CODE REVIEW

File: erroranalysis/requirements.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@ numpy>=1.17.2
 pandas>=0.25.1
 scipy>=1.4.1
 scikit-learn>=0.22.1
-lightgbm>=3.1.1
\ No newline at end of file
+lightgbm>=2.0.11
```
Comment (gaugup) [inline erroranalysis/requirements.txt:5]: Why the magic version 2.0.11? Seems like there are even versions here: https://pypi.org/project/lightgbm/#history?

Comment (romanlutz) [reply erroranalysis/requirements.txt:5]: That's the one required by automl

Comment (romanlutz) [reply erroranalysis/requirements.txt:5]: or rather: the minimum required by automl

**Review by romanlutz — APPROVED** [review]

File: erroranalysis/requirements.txt (hunk 1 above)
Comment (gaugup) [reply erroranalysis/requirements.txt:5]: oh. I thought it was 2.3.0. That is the upper bound.

**Review by gaugup — APPROVED** [review]

**Pull request merged**